### ISSUE TITLE
Add a Spacer reference notebook

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#3415]
For completeness I would recommend adding a `Spacer` Reference Notebook at https://panel.holoviz.org/reference/layouts/Spacer.html

Besides the normal reference stuff, it could also show some use cases to make other layouts span and align things.

**Comment by ahuang11** [comment]
Yep! It's still missing. Any interest in helping contribute a notebook?

I think it'd simply be copying and pasting one of these notebooks and changing it a bit. https://github.com/holoviz/panel/tree/main/examples/reference/layouts
